browser/ui/android/default_browser_promo/java/src/org/chromium/chrome/browser/ui/default_browser_promo/BraveDefaultBrowserPromoUtils.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
```

Comment:
Looks like it's Chromium's header, we have different in our java files.

Reply:
changed it our copyright text